src/clients/SpokePoolClient.ts

```diff
@@ -866,4 +869,19 @@ export class SpokePoolClient extends BaseAbstractClient {
 
     return deposit;
   }
+
+  /**
+   * Determines whether a deposit originates from a lite chain.
+   * @param deposit The deposit to evaluate.
+   * @returns True if the deposit originates from a lite chain, false otherwise. If the hub pool client is not defined,
+   *          this method will return false.
+   */
+  protected doesDepositOriginateFromLiteChain(deposit: DepositWithBlock): boolean {
```

Comment:
can you add a regression test for this function? Also we need one in the sdk if it doesn't exist yet

Reply:
We can't easily add a regression test to this specific function, but the underlying config store function has regression tests

Reply:
why can't you test this function?

Reply:
Tested in follow up commits